scripts/platform-init/platform-init.py

```diff
@@ -86,20 +102,24 @@ def init_secret_service_secret(config: Config):
                 type="Opaque",
             ),
         )
-    elif existing_private_key is None and config.secret_service_private_key is not None:
+    elif config.secret_service_private_key is not None:
```

Comment:
suggestion: We are on python 3.12 in this script. So I think this whole section of if/elif/else should be converted to match/case because it will be easier to follow what case/conditions are being met in each branch.